understanding/20/parsing.html

```diff
@@ -11,12 +11,12 @@ <h1>Understanding Parsing</h1>
    <section id="brief">
 		<h2>In brief</h2>
 		<dl>
-		<dt>Objective</dt><dd>Assistive Technology can properly present page content</dd>
-		<dt>Author task</dt><dd>Create web pages according to specifications</dd>
-		
-		</dl>
-		
-		</section>
+         <dt>Goal</dt><dd>Assistive technology can properly present page content.</dd>
+         <dt>What to do</dt><dd>Create web pages according to specifications.</dd>
+         <dt>Why it's important</dt><dd>People can surf the web with their assistive technology.</dd> 
```

Comment:
@rainbreaw commented
> Should this be changed to
```suggestion
         <dt>Why it's important</dt><dd>People can browse the web with their assistive technology.</dd> 
```
 Or maybe even
```suggestion
         <dt>Why it's important</dt><dd>People can browse and understand web content more easily with their assistive technology.</dd> 
```

Reply:
@GreggVan said
+1 to mbgower comment  (prefer the latter of your two suggestions)

Reply:
I'm inclined to regard "understand" as a bit of an overstatement for the benefits of parsing for people. That to me gets into the Info and Relationships considerations. This is simply making sure _assistive technologies_ can make sense of it, so I'm going to merge the two wording suggestions into
```suggestion
         <dt>Why it's important</dt><dd>People can browse web content more easily with their assistive technology.</dd> 
```